api/inscripciones.js

```diff
@@ -0,0 +1,49 @@
+import { Router } from "express";
+import { Enrollment, Student, Subject } from "../db/index.js";
+import { error } from "./common.js";
+
+const inscripciones = Router()
+
+inscripciones.route("/")
+    .get(async (req, res) => {
+        res.json(await Enrollment.findAll())
+    })
+    .post(async (req, res) => {
+        const { studentID, subjectID } = req.body;
+
+        const student = await Student.findByPk(studentID);
+        const subject = await Subject.findByPk(subjectID);
+
+        if (!student || !subject) {
+            return error(res, 400, "no existe el alumno o la materia"); //se podría sacar, ya tira error cuando no existen
```

Comment:
Usar 404 (la secuela)

```suggestion
            return error(res, 404, "no existe el alumno o la materia"); //se podría sacar, ya tira error cuando no existen
```